### ISSUE TITLE
Loss doesn't converge

### ISSUE THREAD

**Issue by yashkhurana24** [demonsjin/Blind2Unblind#22]
Hello, I'm trying to denoise a 2D image. I add synthetic noise as follows: 
`np.array(np.random.poisson(0.01 * x / 0.1) / 0.01`
But the loss doesn't converge. I have several different learning rates. Can you help me out? 

![Fe K](https://github.com/zejinwang/Blind2Unblind/assets/146676268/9cc9a431-236f-42b8-a56b-39fe062d4450)

**Comment by zejinwang** [comment]
If the training set is small, you'd better set the initial learning rate to 10 times larger. Besides, it is normal that the trend of decreasing loss is not obvious.
